language/move-lang/src/parser/syntax.rs

```diff
@@ -1314,12 +1311,11 @@ fn parse_file<'input>(tokens: &mut Lexer<'input>) -> Result<FileDefinition, Erro
         }
         let function = parse_function_decl(tokens, /* allow_native */ false)?;
         if tokens.peek() != Tok::EOF {
-            return Err(unexpected_token_error(
-                tokens.file_name(),
-                tokens.start_loc(),
-                tokens.content(),
-                vec![],
-            ));
+            let loc = current_token_loc(tokens);
+            return Err(vec![(
+                loc,
+                "Unexpected text after end of main function".to_string(),
```

Comment:
Nit: Would "Unexpected characters..." be better? Text sounds odd and possibly too specific to me.

Reply:
OK. I'm fine with "characters"